_posts/2016-08-17-user-stories.md

```diff
@@ -0,0 +1,88 @@
+
+In order to build a shared understanding about the product vision, goals and values it is necessary to have every team member involved in the process (real users included!). User Stories are a tool that help accomplish that: they improve communication within teams and stakeholders and reduce the risk of misinterpreting product development requirements.
+
+### What is a User Story?
+
+A User Story is a short sentence written from the user’s perspective which describes a valuable functionality our software or system will have.
+
+A User Story is made by 3 elements:
+
+ - Card
+ - Conversation
+ - Confirmation
+
+**The Card** is the written description of the feature to implement. It must be short, 1 or 2 sentences maximum, because it acts as a reminder to discuss the feature during the Conversation phase.
+
+**The Conversation** is the most important element of a User Story. To avoid misinterpretation the Customer Team (Product Manager, Project Manager, Tester, Interaction Designer, Users) and the Developers Team start a discussion around the description to identify its details and ensure that the developer will build the right solution meeting the needs of the user.
+
+**The Confirmation** are the criteria, written as acceptance tests, which determine when a story can be considered "DONE".
+
+A User Story is always written from the perspective of the user. Common description formats that can help teams new to Agile practices as SCRUM are:
+
+> As a < user role > I < want, need, can, etc > < goal > so that < reason >
+
+> In order to < reason > as a < user role > I < want, need, can, etc > < goal > 
+
+The formats help keep the focus on the right questions to ask to reduce the risk of getting into technical details. 
+
+### Who writes User Stories?
+
+Everyone is involved in the creation of user stories: customers, users and developers. The stories go into the Product Backlog and the Product Owner has the responsibility of deciding which one to keep and which one won’t be implemented.
+
+### What makes a good User Story?
+
+The elements of a good User Story can be summarised with the acronym INVEST:
+
+ - Independent
+ - Negotiable
+ - Valuable to users and customers
+ - Estimatable
+ - Small
+ - Testable
+
+User Stories should be **INDEPENDENT** from one another to avoid prioritisation and planning issues. An independent story can be implemented at any time of the or removed from the product backlog without having a negative impact on the other stories. 
```

Comment:
User Stories should be INDEPENDENT **of** one another to avoid prioritisation and planning issues. An independent story can be implemented at any time of the or removed from the product backlog without having a negative impact on the other stories.